ecosystem/sep-0038.md

```diff
@@ -277,7 +295,7 @@ The client must provide either `sell_amount` or `buy_amount`, but not both.
 
 These prices are indicative. The actual price will be calculated at conversion time once the Anchor receives the funds from a User.
 
-The price returned includes any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, amount, delivery method, country code, or other factors.
+The price returned includes any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, amount, delivery method, country code, or other factors. Anchors can optionally detail the costs of the margin in the `fee_details` response field.
```

Comment:
I think we'll have to remove this statement if we use my suggestions. Happy to discuss further.